cypress/integration/elementChecks.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Integartion', () => {
+  describe('First Test', () => {
+    it('Search Component Input Field empty', () => {
+      cy.visit('http://localhost:8080/');
```

Comment:
this line could be moved to `beforeEach`, basically it should be done in commands to avoid copy past

Reply:
Updated